testing/README.md

```diff
@@ -0,0 +1,95 @@
+# How to Test E2E Resource Changes
+This guide will give a step by step instruction on how to test changes made to E2E testing resources before pushing a PR. 
+
+### 1. Create an IAM Role with OIDC Identity Provider
+This step is needed to allow Github Action to have access to resources in the AWS account
+#### Create an OIDC Provider
+- First step is to create an OIDC Identity Provider to allow Github action access to the AWS account resource. Login to AWS, go to the IAM console and click on the Identity Providers tab.
+- Click on Add Provider, choose OpenID Connect and type `https://token.actions.githubusercontent.com` in the Provider URL. Click "Get thumbprint". For Audience, use `sts.amazonaws.com`. Finally, click "Add provider"
+#### Create an IAM role
+- Next, an IAM role needs to be created using the OIDC Identity Provider. Go to the Roles tab and click Create role. 
+- Choose Web Identity, and choose `token.actions.githubusercontent.com` as the Identity provider, Audience as `sts.amazonaws.com`, and for Github organizations put your github username down. Click next.
+- Add the AdministratorAccess policy. Click next.
+- Enter your Role name. Click "Create role".
```

Comment:
Can't we do this programatically?

Reply:
We can. We also have to enter the console anyways to manually change the edit policy for the role so I think we can just make the programmer do this on the console?